core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -89,6 +94,18 @@ func (env *LevelDBTestEnv) GetName() string {
 	return "levelDBTestEnv"
 }
 
+// DBValueFormat returns the format used by the stateleveldb for dbvalue
+func (env *LevelDBTestEnv) DBValueFormat() byte {
+	return stateleveldb.TestEnvDBValueformat
+}
+
+// DecoDecodeDBValue decodes the dbvalue bytes for tests
```

Comment:
```suggestion
// DecodeDBValue decodes the dbvalue bytes for tests
```

Reply:
Not sure how linter passed this - will fix